src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import classNames from "classnames";
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const currentTabId = tabs.find(tab => tab.id === activeTabId)
+    ? activeTabId
+    : tabs[0].id;
+
+  const currentTab = tabs.find(tab => tab.id === currentTabId);
```

Comment:
you can simplify this logic
```suggestion
    const currentTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];

```